recipes/deepdiff/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "deepdiff" %}
+{% set version = "3.3.0" %}
+{% set md5 = "d16022fbf7b6a1ad401c629b81088614" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - idna 2.5     # [win]
```

Comment:
I don't think `idna` or  `jsonpickle` are required to actually  build the package.